docusaurus/docs/dev-docs/plugins/graphql.md

```diff
@@ -628,7 +628,7 @@ export default {
 
 ##### Policies
 
-[Policies](/dev-docs/backend-customization/policies) can be applied to a GraphQL resolver through the `resolversConfig.[MyResolverName].policies` key.
+[Policies](/dev-docs/backend-customization/policies) can be applied to a GraphQL resolver through the `resolversConfig.[MyResolverName].policies` key. Only difference between the GraphQL and REST implementation is that the `config` key becomes `options`.
```

Comment:
Where did you encounter `options`? :thinking: 
GraphQL => https://github.com/strapi/strapi/blob/main/packages/plugins/graphql/server/services/content-api/policy.js#L18
REST => https://github.com/strapi/strapi/blob/main/packages/core/strapi/lib/services/server/policy.js#L16

Reply:
@Convly It is for middleware
https://github.com/strapi/strapi/blob/6c81e74ebc7b39f541e864a92907bc530a48c842/packages/plugins/graphql/server/services/content-api/wrap-resolvers.js#L38-L40C38
Made a mistake that I thought it where policies.